src/storage/mvcc/txn.rs

```diff
@@ -458,6 +458,22 @@ impl<S: Snapshot> MvccTxn<S> {
                         pessimistic: true,
                     });
                 }
+                // A lock with larger min_commit_ts than current commit_ts can't be committed
+                if commit_ts < lock.min_commit_ts {
```

Comment:
What if the cluster is upgraded from an old version and its `min_commit_ts` is 0?